test-apps/imodel-transformer/src/Main.ts

```diff
@@ -165,6 +165,20 @@ void (async () => {
           type: "boolean",
           default: false,
         },
+        isolateElements: {
+          desc: "transform filtering all unnecessary element/model trees except for those listed in a comma-separated argument of ids",
+          type: "string",
+        },
+        loadSourceGeometry: {
+          desc: "load geometry from the source as JSON while transforming, for easier (but not performant) transforming of geometry",
+          type: "boolean",
+          default: false,
+        },
+        cloneUsingJsonGeometry: {
+          desc: "clone using json geometry",
```

Comment:
can you give more helpful info in the description, this just repeats the parameter name with spaces

Reply:
whoops I meant to use a better description but wasn't sure if I was going to keep this one. Will fix.

Reply:
is that better?